distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java

```diff
@@ -276,20 +364,24 @@ protected UrlResourceReaderConfigurator getUrlResourceReaderConfigurator() {
                 editConfigurationFilePut("etc/system.properties", "eclipse.enableStateSaver",
                         Boolean.FALSE.toString()),
 
-                editConfigurationFilePut("etc/org.apache.karaf.shell.cfg", "sshPort", SSH_PORT),
-                editConfigurationFilePut("etc/ddf.platform.config.cfg", "port", HTTPS_PORT),
+                editConfigurationFilePut("etc/org.apache.karaf.shell.cfg", "sshPort",
+                        Port.SSH_PORT.getPort(basePort)),
+                editConfigurationFilePut("etc/ddf.platform.config.cfg", "port",
+                        Port.HTTPS_PORT.getPort(basePort)),
                 editConfigurationFilePut("etc/ddf.platform.config.cfg", "host", "localhost"),
                 editConfigurationFilePut("etc/org.ops4j.pax.web.cfg", "org.osgi.service.http.port",
-                        HTTP_PORT), editConfigurationFilePut("etc/org.ops4j.pax.web.cfg",
-                        "org.osgi.service.http.port.secure", HTTPS_PORT),
+                        Port.HTTP_PORT.getPort(basePort)),
+                editConfigurationFilePut("etc/org.ops4j.pax.web.cfg",
+                        "org.osgi.service.http.port.secure", Port.HTTPS_PORT.getPort(basePort)),
                 editConfigurationFilePut("etc/org.apache.karaf.management.cfg", "rmiRegistryPort",
-                        RMI_REG_PORT),
+                        Port.RMI_REG_PORT.getPort(basePort)),
                 editConfigurationFilePut("etc/org.apache.karaf.management.cfg", "rmiServerPort",
-                        RMI_SERVER_PORT), replaceConfigurationFile("etc/hazelcast.xml",
+                        Port.RMI_SERVER_PORT.getPort(basePort)),
+                replaceConfigurationFile("etc/hazelcast.xml",
                         new File(this.getClass().getResource("/hazelcast.xml").toURI())),
-                replaceConfigurationFile("etc/ddf.security.sts.client.configuration.cfg", new File(
-                        this.getClass().getResource("/ddf.security.sts.client.configuration.cfg")
-                                .toURI())), replaceConfigurationFile(
+                editConfigurationFilePut("etc/ddf.security.sts.client.configuration.cfg", "address",
```

Comment:
Why did this change from a full replace of the sts client config file to an edit?